Application/People/Group/Service/Data.php

```diff
@@ -51,10 +51,11 @@ public function setupDatabaseContent()
      * @param string $Remark
      * @param bool   $IsLocked
      * @param string $MetaTable
+     * @param bool   $IsCoreGroup
      *
      * @return TblGroup
      */
-    public function createGroup($Name, $Description, $Remark, $IsLocked = false, $MetaTable = '')
+    public function createGroup($Name, $Description, $Remark, $IsLocked = false, $MetaTable = '', $IsCoreGroup = false)
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$IsLocked" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)